### DISCUSSION TITLE
Is it intended that malt be run as root for Linux?

### DISCUSSION THREAD

**Discussion by Zeggpold87** [bnpr/Malt#67] [Q&A] [answered]
Using blender 2.92 on Gentoo Linux, I get a hang when selecting Malt as a renderer.  Looking at the Malt log file, it looks like it's getting a permission denied error when trying to raise the priority (a negative nice value) and crashing, then restarting only to crash again.  (If this isn't intended, I can file a bug with the log file.)

**Comment by pragma37** [comment, marked as answer]
Certainly not intended.  Please, open a new issue so we can solve it there.

**Reply by Zeggpold87** [reply]
Done, it's issue #68.  Thanks for the quick response.

**Reply by Zeggpold87** [reply]
I saw your reply in the issue; thanks again for the quick reply!  But how do I test the "latest build"?  The release doesn't appear to have changed, so I guess I need to pull from git, but then what do I need to do?  The release file seems to have a different structure than what I pulled from git.  Sorry for all the questions, I did look at all the docs I could find but I'm not really familiar with blender add-ons.

**Reply by pragma37** [reply]
You can always download the latest build in the Release section:
https://github.com/bnpr/Malt/releases

A new release is generated automatically every time the code changes, so it's always up to date.
